src/identify/grid.rs

```diff
@@ -248,16 +248,18 @@ fn find_alignment_pattern<S>(img: &mut PreparedImage<S>, mut align_seed: Point,
             let y = align_seed.y as usize;
 
             // Alignment pattern should not be white
-            if PixelColor::White != img.get_pixel_at(x, y) {
-                let region = img.get_region((x, y));
-                let count = match region {
-                    ColoredRegion::Unclaimed { pixel_count, .. } => { pixel_count }
-                    _ => continue,
-                };
-
-                // Matches expected size of alignment pattern
-                if count >= size_estimate / 2 && count <= size_estimate * 2 {
-                    return Some(align_seed);
+            if x < img.width() && y < img.height() {
```

Comment:
I prefer using
```
if x >= img.width() || y >= img.height() {
   continue
}
```

I don't like deeply nested code, and looking back at this code, it should even be changed to

```
if img.get_pixel_at(x, y) == PixelColor::White {
  continue
}
```

Reply:
This would change the semantics since the `align_seed` `Point` gets modified further bellow.

Reply:
I see. That code I copied from C is a mess 🙃